opennms-bootstrap/src/main/java/org/opennms/bootstrap/Bootstrap.java

```diff
@@ -395,6 +395,8 @@ private static void configureRMI(final ClassLoader cl) throws IOException {
             m_rmiServerSocketFactory = new HostRMIServerSocketFactory("localhost");
             RMISocketFactory.setSocketFactory(m_rmiServerSocketFactory);
         }
+
+        OpenNMSProxyLoginModule.setClassloader(cl);
```

Comment:
Please add a descriptive comment here about why this is necessary so that people reading or updating the code understand this.